### ISSUE TITLE
Packaged in Fedora?

### ISSUE THREAD

**Issue by brianjmurrell** [fedora-infra/anitya#1259]
Is any of this packaged in Fedora anywhere?  In particular, given that `fedora-messaging` consumers might consume anitya messages, [anitya_schema/project_messages.py](https://github.com/fedora-infra/anitya/blob/master/anitya_schema/anitya_schema/project_messages.py) would be useful on systems running fedora-messaging for use in their `callback`.

**Comment by Zlopez** [comment]
Unfortunately not. But if anybody wants to package it and maintain it they are free to do it.

**Comment by StephenCoady** [comment]
We are currently working on automating the packaging of all of our infrastructure applications. I've created #1346 to track this work for anitya on our board.

**Comment by Zlopez** [comment]
We should probably separate the messaging scheme from the repository to package it as a separate package. I would like to move it to completely separated repository, but I didn't get to it yet.

**Comment by StephenCoady** [comment]
If someone wants to create the repo and move what needs to be moved then we will for sure add it to packit.

**Comment by erolkskn** [comment]
Is this issue still relevant ? I would love to work on that.

**Comment by Zlopez** [comment]
Yes, there is still not any package in Fedora. Also the message schema is now a separate repository and could be found [here](https://github.com/fedora-infra/anitya-messages).

**Comment by romulasry** [comment]
It seems all ready now, someone just has to make the package on fedora, see: https://release-monitoring.org/project/322887/

**Comment by Zlopez** [comment]
If anybody wants to package Anitya in Fedora they are welcome to do it. I can help if needed.

**Comment by erolkskn** [comment]
@Zlopez Yeah I was going to work on that but then I forgot. Anyway, this should be packaged using Packit right ?
